flink-tests/src/test/java/org/apache/flink/test/exampleJavaPrograms/WordCountITCase.java

```diff
@@ -27,7 +27,6 @@
 	protected String textPath;
 	protected String resultPath;
 
-
```

Comment:
I know we should refrain from whitespace reformats. It was accidental, but I think it's OK (two empty lines). I agree that we shouldn't do whitespace reformats, but I don't think that these are affecting review-ability of this PR.